xikolo-ios/ViewControllers/Courses/CourseDecisionViewController.swift

```diff
@@ -10,8 +10,9 @@ class CourseDecisionViewController: UIViewController {
     enum CourseContent: Int {
         case learnings = 0
         case discussions = 1
-        case courseDetails = 2
-        case announcements = 3
+        case certificates = 2
```

Comment:
Please add the certificates as the last item

Reply:
Note that this doesn't correspond to the felt order of the website. In the dashboard it's before announcements. But I'll do it.